From 973d16cc479e8962f217b2fc89a1d34d8e144102 Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Thu, 27 Aug 2020 14:53:53 +0100
Subject: [PATCH 3/5] efi: change usage syntax

syntax was changed from "," to ":". Account this also
in the documentation.
Fixes: 2b3fbcb59f efi_loader: use ':' as separator for setenv -i

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
---
 cmd/nvedit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/nvedit.c b/cmd/nvedit.c
index d188c6aa6b..d03accead9 100644
--- a/cmd/nvedit.c
+++ b/cmd/nvedit.c
@@ -1472,7 +1472,7 @@ static char env_help_text[] =
 	"env select [target] - select environment target\n"
 #endif
 #if defined(CONFIG_CMD_NVEDIT_EFI)
-	"env set -e [-nv][-bs][-rt][-at][-a][-i addr,size][-v] name [arg ...]\n"
+	"env set -e [-nv][-bs][-rt][-at][-a][-i addr:size][-v] name [arg ...]\n"
 	"    - set UEFI variable; unset if '-i' or 'arg' not specified\n"
 #endif
 	"env set [-f] name [arg ...]\n";
-- 
2.17.1

